metric-publishers/emf-metric-publisher/src/test/java/software/amazon/awssdk/metrics/publishers/emf/internal/MetricEmfConverterTest.java

```diff
@@ -165,4 +166,16 @@ void ConvertMetricCollectionToEMF_OverSizedRecords(){
 
     }
 
+    @Test
+    void ConvertMetricCollectionToEMF_LargeCollection(){
```

Comment:
Let's be more descriptive on the naming following `methodToTest_when_should`, suggesting `ConvertMetricCollectionToEMF_metricCollectionExceedsMaxSize_shouldSplit`